src/psrp/_pwsh/copy.ps1

```diff
@@ -36,6 +40,9 @@ begin {
     if (-not (Test-Path -LiteralPath $parentDir)) {
         throw "Target path directory '$parentDir' does not exist"
     }
+    if (!$overwrite -and (Test-Path -LiteralPath $Path)) {
```

Comment:
Just because my profile will auto format this to `-not` can you make this change please?

```suggestion
    if (-not $overwrite -and (Test-Path -LiteralPath $Path)) {
```